_layouts/restdocs.html

```diff
@@ -0,0 +1,27 @@
+---
+layout: default
+---
+<!DOCTYPE html>
```

Comment:
This is added within the context of the default template, that is: we now have an embedded <html><head> etc within an HTML document. I believe all you need to add is the redoc and script tags.